W3CTRMANIFEST

```diff
@@ -0,0 +1,5 @@
+webrtc.html?shortName=webrtc respec
```

Comment:
Who reads this file? Is it possible to describe that either in a comment or in README?